packages/angular/src/schematics/init/schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "ng-add",
+  "title": "ng-add",
+  "type": "object",
+  "properties": {
+    "project": {
+      "type": "string",
+      "description": "The name of the project to add Testronaut to",
+      "$default": {
+        "$source": "argv",
+        "index": 0
+      },
+      "default": ""
+    },
+    "createExamples": {
```

Comment:
what about `generateExamples` or `withExamples`?

Reply:
As suggested I renamed to `withExamples`